runtime/jcl/exports.cmake

```diff
@@ -688,3 +688,10 @@ if(NOT JAVA_SPEC_VERSION LESS 19)
 		Java_jdk_internal_vm_Continuation_unpin
 	)
 endif()
+
+# Java 20+
```

Comment:
nit: it is `# java 20+` in https://github.com/eclipse-openj9/openj9/pull/16574

Reply:
This is updated to `'Java 20+` as requested at https://github.com/eclipse-openj9/openj9/pull/16585#discussion_r1082847896.

Reply:
I should update this with `java 20+` to keep consistent with the rest of the script in terms of java version.

Reply:
Restored to `java 20+`.

Reply:
`Java` should be capitalized since it is a name. The incorrect lower-case instances should be fixed; but, they do not need to be fixed in this PR.

Reply:
Updated with `Java 20` in my changes at https://github.com/eclipse-openj9/openj9/pull/16585/commits/16c03d508fba1962af68b6c713aaa938720e60cf#diff-8e3f29d0e205cd3f6cfd36ef0b25a9a1d2df93d47a86edf81fdf27763b97cb13.